eras/conway/impl/CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
   * `reCommitteeStateL`
 * Add a new field to `GovInfoEvent` and change "unclaimed" field from `Set` to a `Map`.
 * Changed return type of `proposalsShowDebug`
+* Add `TxInfo` translation golden test. #4629
```

Comment:
Addition of tests is not a user facing feature, so they should never be mentioned in the changelog. 

Changelog is there for people who use this package as a library, so the can easily identify changes that affect them. 

That being said. Considering that #4599 added a `gen-golden` executable, which is a publicly visible change, we can keep this changelog entry, but it should have been added in #4599 and it should have read something like:

```suggestion
* Added `gen-golden` executable needed for golden tests: #4629
```